OasisDEX/oasis-borrow
# Lw/7315 unfollow api

# [Unfollow endpoint](https://app.shortcut.com/oazo-apps/story/6664/create-api-endpoints-for-followed-vaults-following)
# [Unfollow button handling](https://app.shortcut.com/oazo-apps/story/7316/unfollow-vault-using-ui-api)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>
- Added unfollow endpoint
- Handled unfollow button
  
## How to test 🧪
Open vault that exist in database
- Follow vault, there's no record in db
![image](https://user-images.githubusercontent.com/6871923/212949874-71bce217-a5b1-4bee-9ddd-3a90c9cb5822.png)
- After following button should change to Following state
![image](https://user-images.githubusercontent.com/6871923/212950521-6b1427b1-7180-4791-86d7-1057562f0013.png)

- Unfollow vault that is followed in db
![image](https://user-images.githubusercontent.com/6871923/212950277-aaaa815c-a9ac-4298-9067-17fca233d1c0.png)

- When Unfollowing vault should be processing for a moment, but then changed back to Follow
![image](https://user-images.githubusercontent.com/6871923/212950876-72e6d687-5ccc-4976-9bd1-4808c77433c7.png)

![image](https://user-images.githubusercontent.com/6871923/212951012-02761914-d26d-44bb-807e-3731b5547c33.png)
